wpilibc/src/main/native/include/frc/Watchdog.h

```diff
@@ -67,8 +64,10 @@ class Watchdog {
 
   ~Watchdog();
 
-  Watchdog(Watchdog&&) = default;
-  Watchdog& operator=(Watchdog&&) = default;
+  Watchdog(const Watchdog&) = delete;
+  Watchdog& operator=(const Watchdog&) = delete;
```

Comment:
Are these not implicitly deleted already?

Reply:
Oh, yes, they would be, due to the user-defined move ones.